MovingBlocks/CrashReporter
# fix(#4699): CrashReporter upload issues

_Note for the Reviewer: I encourage to review this PR commit by commit._

This PR
- fixes the broken pastebin integration
- fixes the broken GDrive integration
- shares reports uploaded to GDrive with the terasology@gmail.com GDrive
- replaces IRC with Discord
- upgrades the CrashReporter gradle version
- updates the copyrights

How to test:
1. Get the CrashReporter lib via `./groovyw lib get CrashReporter`
2. In `lib/CrashReporter` check out this PR
3. In the workspace root, check out the [engine PR #4844](https://github.com/MovingBlocks/Terasology/pull/4844)
4. Run Terasology from source (there's a runconfig "with CR" in intellj, but this didn't work for me so I did it via the terminal)
5. Provoke a crash (I did this by selecting the `AlchemyPlantGenerator` module and attempting to start a world)
6. Check that in the CrashReporter upload screen you can upload both, to PasteBin and Google Drive
7. Check that in the final CrashReporter screen the Discord link works and you don't see an option to join IRC anymore.

Depends on https://github.com/MovingBlocks/Terasology/pull/4844

Fixes https://github.com/MovingBlocks/Terasology/issues/4699
